packages/lib/src/components/PayPal/utils.ts

```diff
@@ -19,7 +19,7 @@ const getPaypalSettings = ({ amount, countryCode, debug, environment = '', local
     const isTestEnvironment: boolean = environment.toLowerCase() === 'test';
     const clientId: string = isTestEnvironment ? ADYEN_CLIENTID_TEST : ADYEN_CLIENTID_LIVE;
 
-    const { merchantId, intent } = configuration;
+    const { merchantId, intent, commit } = configuration;
```

Comment:
The `commit` property will arribe inside the `props`, not this `configuration` object.